buildSrc/src/main/kotlin/com/datadog/gradle/plugin/jsonschema/GenerateJsonSchemaTask.kt

```diff
@@ -97,13 +74,37 @@ open class GenerateJsonSchemaTask : DefaultTask() {
         val topDir = getOutputDir()
         val outputPackageDir = File(
             topDir.absolutePath + File.separator +
-                extension.targetPackageName.replace('.', File.separatorChar)
+                targetPackageName.replace('.', File.separatorChar)
         )
         return outputPackageDir
     }
 
-    internal fun setParams(extension: JsonSchemaExtension) {
-        this.extension = extension
+    // endregion
+
+    // region Task action
+
+    /**
+     * The main [TaskAction].
+     */
+    @TaskAction
+    fun performTask() {
+        val inputDir = getInputDir()
+        val outputDir = getOutputDir()
+        val files = getInputFiles()
+            .filter { it.name !in ignoredFiles }
+
+        println("Found ${files.size} in input dir: $inputDir")
```

Comment:
probably word `files` is missing in this output? Also I'm just curious why `println` is used instead of Gradle's `logger`

Reply:
hmmm....what's the Gradle logger doing  extra ?

Reply:
almost. just `println` is `QUIET` level by default, while `logger` gives control on the level.